src/v2/db_query.py

```diff
@@ -0,0 +1,199 @@
+            Year, Genres, Num_Votes, Release_Date, Directors) 
+            VALUES(%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s);""") 
+            values=[row['Const'], row['Your Rating'], row['Date Rated'], row['Title'], row['URL'], 
+            row['Title Type'], row['IMDb Rating'], row['Runtime (mins)'], 
+            row['Year'], row['Genres'], row['Num Votes'], row['Release Date'], row['Directors']]
+            self.cursor.execute(columns, values)
+            self.conn.commit()
+        
+        except (Exception, ps.DatabaseError) as error:
+            print(error)
+
+    def read(self, query, params):
+        '''
+        Summary Line 
+        Extended Description of read(self, query, params)
+
+        parameters:
+        ----------
+        query 
+        params
+
+        executes sql query along with params if exists
+        prints the status after each execution
+
+        raises exception if any
+        '''
+        try:
+            values =[params]
+            self.cursor.execute(query, values)
+            result=self.cursor.fetchall()
+            print(result)
+            print("\n")
+        except (Exception, ps.DatabaseError) as error:
+            print(error)
+
+    def update(self, const, new_rating):
+        '''
+        Summary Line
+        Extended description of update(self, const, new_rating)
+
+        parameters:
+        -----------
+        const //primary key
+        new_rating
+
+        selects record in ratings table that matches const
+
+        checks the validity of the new_rating 
+        proceeds with updation only if values ranging between 0 and 10
+        
+        updates the value of column your_rating with new_rating
+        also updates date_rated with current date
+        prints the status
+
+        raises exception if any
+        '''
+        if (new_rating>0.0 and new_rating<10.0):
+
+            day=datetime.now()
+            day=day.strftime("%d/%m/%Y")
+            try:
+                query = '''UPDATE ratings SET Your_Rating = %s, Date_Rated = %s
+                WHERE Const = %s;'''
+                values=[new_rating, day, const]
+                self.cursor.execute(query, values)    
+                self.conn.commit()
+                print("values updated in the database successfully...\n")
+
+            except (Exception, ps.DatabaseError) as error:
+                print(error)
+        else:
+            sys.exit("enter a valid value between 0 and 10!!\n")
+        
+    def delete(self, const):
+        '''
+        Summary Line 
+        Extended Description of delete(self, const)
+
+        parameters:
+        -----------
+        const //primary key
+
+        selects the rating in the record that matches with const
+        sets the rating to 0
+        prints the status
+
+        raises exception if any
+        '''
+        try:
+            query='''UPDATE ratings SET Your_Rating = %s WHERE Const = %s;'''
```

Comment:
This is not a DELETE statement. The table will still show a rating for that particular title exists. Change to delete statement